File: sdks/go/pkg/models/shared/retentionpolicies.go
package shared

// RetentionPolicies
// The part of an API response that describes marker
// based pagination
type RetentionPolicies struct {
	Entries    []RetentionPolicyMini `json:"entries,omitempty"`
	Limit      *int64                `json:"limit,omitempty"`
	NextMarker *string               `json:"next_marker,omitempty"`
}

File: sdks/ts/src/sdk/models/shared/retentionpolicies.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { RetentionPolicyMini } from "./retentionpolicymini";



// RetentionPolicies
/** 
 * The part of an API response that describes marker
 * based pagination
**/
export class RetentionPolicies extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=entries", elemType: RetentionPolicyMini })
  entries?: RetentionPolicyMini[];

  @SpeakeasyMetadata({ data: "json, name=limit" })
  limit?: number;

  @SpeakeasyMetadata({ data: "json, name=next_marker" })
  nextMarker?: string;
}
